src/makeCalculator.js

```diff
@@ -4,7 +4,31 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculator = {};
+
+  calculator.result = 0;
+
+  calculator.operate = (callback, number) => {
+    calculator.result = callback(number);
+
+    return calculator;
+  };
+
+  calculator.add = (a) => calculator.result + a;
+
+  calculator.subtract = (a) => calculator.result - a;
+
+  calculator.multiply = (a) => calculator.result * a;
+
+  calculator.divide = (a) => calculator.result / a;
+
+  calculator.reset = () => {
+    calculator.result = 0;
+
+    return calculator;
+  };
```

Comment:
Code knowledge issue: You should not use the `function` keyword when defining methods in an object. Instead, you should use the shorthand method syntax. For example, instead of writing `reset: function() {...}`, you should write `reset() {...}`. This is a more modern and concise syntax.